[2025-02-23T15:52:37.277+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-23T15:52:37.313+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_first_dag.first_task scheduled__2025-02-11T00:00:00+00:00 [queued]>
[2025-02-23T15:52:37.329+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_first_dag.first_task scheduled__2025-02-11T00:00:00+00:00 [queued]>
[2025-02-23T15:52:37.330+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 6
[2025-02-23T15:52:37.453+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): first_task> on 2025-02-11 00:00:00+00:00
[2025-02-23T15:52:37.463+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'my_first_dag', 'first_task', 'scheduled__2025-02-11T00:00:00+00:00', '--job-id', '17', '--raw', '--subdir', 'DAGS_FOLDER/my_first_dags.py', '--cfg-path', '/tmp/tmp1tzwoahj']
[2025-02-23T15:52:37.467+0000] {standard_task_runner.py:105} INFO - Job 17: Subtask first_task
[2025-02-23T15:52:37.472+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1387) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-23T15:52:37.475+0000] {standard_task_runner.py:72} INFO - Started process 1402 to run task
[2025-02-23T15:52:37.546+0000] {task_command.py:467} INFO - Running <TaskInstance: my_first_dag.first_task scheduled__2025-02-11T00:00:00+00:00 [running]> on host 2cebbf768e94
[2025-02-23T15:52:37.634+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='gzang' AIRFLOW_CTX_DAG_ID='my_first_dag' AIRFLOW_CTX_TASK_ID='first_task' AIRFLOW_CTX_EXECUTION_DATE='2025-02-11T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-11T00:00:00+00:00'
[2025-02-23T15:52:37.635+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-23T15:52:37.655+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-02-23T15:52:37.658+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hello world! the first task is defined!!!']
[2025-02-23T15:52:37.677+0000] {subprocess.py:99} INFO - Output:
[2025-02-23T15:52:37.678+0000] {subprocess.py:106} INFO - hello world! the first task is defined!!!
[2025-02-23T15:52:37.679+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-02-23T15:52:37.725+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-23T15:52:37.734+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=my_first_dag, task_id=first_task, run_id=scheduled__2025-02-11T00:00:00+00:00, execution_date=20250211T000000, start_date=20250223T155237, end_date=20250223T155237
[2025-02-23T15:52:37.972+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-23T15:52:38.020+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-23T15:52:38.024+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-02-24T04:47:14.118+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-24T04:47:14.150+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_first_dag.first_task scheduled__2025-02-11T00:00:00+00:00 [queued]>
[2025-02-24T04:47:14.162+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_first_dag.first_task scheduled__2025-02-11T00:00:00+00:00 [queued]>
[2025-02-24T04:47:14.163+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 6
[2025-02-24T04:47:14.302+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): first_task> on 2025-02-11 00:00:00+00:00
[2025-02-24T04:47:14.314+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'my_first_dag', 'first_task', 'scheduled__2025-02-11T00:00:00+00:00', '--job-id', '23', '--raw', '--subdir', 'DAGS_FOLDER/my_first_dags.py', '--cfg-path', '/tmp/tmply5cawn_']
[2025-02-24T04:47:14.317+0000] {standard_task_runner.py:105} INFO - Job 23: Subtask first_task
[2025-02-24T04:47:14.318+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=863) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-24T04:47:14.319+0000] {standard_task_runner.py:72} INFO - Started process 883 to run task
[2025-02-24T04:47:14.398+0000] {task_command.py:467} INFO - Running <TaskInstance: my_first_dag.first_task scheduled__2025-02-11T00:00:00+00:00 [running]> on host 79e0cb3ed452
[2025-02-24T04:47:14.556+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='gzang' AIRFLOW_CTX_DAG_ID='my_first_dag' AIRFLOW_CTX_TASK_ID='first_task' AIRFLOW_CTX_EXECUTION_DATE='2025-02-11T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-11T00:00:00+00:00'
[2025-02-24T04:47:14.559+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-24T04:47:14.577+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-02-24T04:47:14.579+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hello world! the first task is defined!!!']
[2025-02-24T04:47:14.595+0000] {subprocess.py:99} INFO - Output:
[2025-02-24T04:47:14.599+0000] {subprocess.py:106} INFO - hello world! the first task is defined!!!
[2025-02-24T04:47:14.599+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-02-24T04:47:14.634+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-24T04:47:14.635+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=my_first_dag, task_id=first_task, run_id=scheduled__2025-02-11T00:00:00+00:00, execution_date=20250211T000000, start_date=20250224T044714, end_date=20250224T044714
[2025-02-24T04:47:14.712+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-24T04:47:14.780+0000] {taskinstance.py:3901} INFO - 2 downstream tasks scheduled from follow-on schedule check
[2025-02-24T04:47:14.794+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
